application/management/commands/helsinki_activities_test_data.py

```diff
@@ -54,6 +54,16 @@
 }
 
 LANDING_PAGE_BASE = {
+    "meta_information_en": "We put together the best foliage for the fall",
+    "meta_information_fi": "Kokosimme parhaat tärpit syksylle",
+    "meta_information_sv": "Vi sätter ihop det bästa bladverket för hösten",
+
+    "page_title_en": "We put together the best foliage for the fall",
+    "page_title_fi": "Kokosimme parhaat tärpit syksylle",
+    "page_title_sv": "Vi sätter ihop det bästa bladverket för hösten",
+}
+
```

Comment:
Until the 2nd part of this PR is ready, `LANDING_PAGE_BASE` still needs `button_text`, `button_url`, and `title` set of fields.

On an empty db, in local dev env, with the "check thing" in `migrate_banner_data` I mentioned in the other comment, trying to populate the db with test data by setting the `HELSINKI_ACTIVITIES_TEST_DATA=true` gives the following error:

```python
api_1       |   File "/app/application/management/commands/helsinki_activities_test_data.py", line 173, in handle
api_1       |     landing_pages_folder.add_child(instance=models.LandingPages(
api_1       |   File "/usr/local/lib/python3.8/site-packages/treebeard/mp_tree.py", line 1022, in add_child
api_1       |     return MP_AddChildHandler(self, **kwargs).process()
api_1       |   File "/usr/local/lib/python3.8/site-packages/treebeard/mp_tree.py", line 389, in process
api_1       |     newobj.save()
api_1       |   File "/usr/local/lib/python3.8/contextlib.py", line 75, in inner
api_1       |     return func(*args, **kwds)
api_1       |   File "/usr/local/lib/python3.8/site-packages/wagtail/core/models.py", line 630, in save
api_1       |     self.full_clean()
api_1       |   File "/usr/local/lib/python3.8/site-packages/wagtail/core/models.py", line 604, in full_clean
api_1       |     super().full_clean(*args, **kwargs)
api_1       |   File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 1203, in full_clean
api_1       |     raise ValidationError(errors)
api_1       | django.core.exceptions.ValidationError: {'button_text_fi': ['This field cannot be blank.'], 'button_text_sv': ['This field cannot be blank.'], 'button_text_en': ['This field cannot be blank.'], 'button_url_fi': ['This field cannot be blank.'], 'button_url_sv': ['This field cannot be blank.'], 'button_url_en': ['This field cannot be blank.'], 'title_fi': ['This field cannot be blank.'], 'title_sv': ['This field cannot be blank.'], 'title_en': ['This field cannot be blank.']}
events-helsinki-cms_api_1 exited with code 1
```

Reply:
Fixed this! 👍